packages/react/src/patterns/blocks/ContentGroupPictograms/__stories__/ContentGroupPictograms.stories.js

```diff
@@ -61,65 +49,50 @@ storiesOf('Patterns (Blocks)|ContentGroupPictograms', module)
-          'Item 1 Heading (required)',
-          'Aliquam condimentum interdum'
-        ),
-        copy: text(
-          'Item 1 Copy (required)',
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.'
-        ),
-        cta: object('Item 1 Link', {
-          type: 'local',
-          href: 'https://www.example.com',
-          copy: 'Lorem ipsum dolor',
-        }),
-        pictogram: {
-          src: selectPictogram(pictogram1),
-          'aria-label': text('Aria-label 1:', 'Desktop'),
-        },
-      },
-      {
-        heading: text(
-          'Item 2 Heading (required)',
-          'Aliquam condimentum interdum'
-        ),
-        copy: text(
-          'Item 2 Copy (required)',
-          'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.'
-        ),
-        cta: object('Item 2 Link', {
+    /**
+     * Toggles CTA data based on item value
+     *
+     * @param {boolean} item defines if cta will be rendered
+     * @returns {*} if true returns cta data, if false, returns null
+     */
+    const toggleCTA = item => {
+      if (item) {
+        return {
           type: 'local',
           href: 'https://www.example.com',
           copy: 'Lorem ipsum dolor',
-        }),
-        pictogram: {
-          src: selectPictogram(pictogram2),
-          'aria-label': text('Aria-label 2:', 'Touch'),
-        },
-      },
-      {
+        };
+      } else {
+        return null;
+      }
+    };
+
+    const pictogramCount = number('Number of PictogramItems', 3);
+    const items = [];
+
+    for (let i = 0; i < pictogramCount; i++) {
+      items.push({
         heading: text(
-          'Item 3 Heading (required)',
+          `Item ${i + 1} Heading (required)`,
           'Aliquam condimentum interdum'
         ),
         copy: text(
-          'Item 3 Copy (required)',
+          `Item ${i + 1} Copy (required)`,
           'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.'
         ),
-        cta: object('Item 3 Link', {
-          type: 'local',
-          href: 'https://www.example.com',
-          copy: 'Lorem ipsum dolor',
-        }),
+        cta: toggleCTA(boolean(`Item ${i + 1} CTA`, false)),
         pictogram: {
-          src: selectPictogram(pictogram3),
-          'aria-label': text('Aria-label 3:', 'Pattern'),
+          src: selectPictogram(
+            select(
+              `Item ${i + 1} Pictogram (required)`,
+              pictograms,
+              pictograms.Desktop
+            )
+          ),
+          'aria-label': text('Aria-label 1:', 'Desktop'),
```

Comment:
I wouldn't add the aria label as a storybook knob. It just makes it harder to see and update what's going on.

Reply:
I've removed it from the story